application/resources/.eslintrc.json

```diff
@@ -0,0 +1,16 @@
+{
+  "rules": {
+    "strict": [
+      "error",
+      "never"
+    ]
+  },
+  "globals": {
+    "BigInt": true,
+    "application": true,
+    "context": true,
+    "api": true,
+    "gs": false,
+    "Duration": false
```

Comment:
Ditto.
Also, these are very similar to each other, I think it would be better to write one `config-eslitrc.json` and then 'extends' it everywhere needed.